{
  "cveId": "CVE-2024-31919",
  "title": "IBM MQ denial of service",
  "description": "IBM MQ 9.0 LTS, 9.1 LTS, 9.2 LTS, 9.3 LTS and 9.3 CD, in certain configurations, is vulnerable to a denial of service attack caused by an error processing messages when an API Exit using MQBUFMH is used.  IBM X-Force ID:  290259.",
  "publishedDate": "2024-06-28T17:34:15.469000+00:00",
  "lastModifiedDate": "2024-08-02T01:59:50.566000+00:00",
  "severity": "MEDIUM",
  "cvssMetrics": [
    {
      "version": "3.1",
      "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "baseScore": 5.9,
      "baseSeverity": "MEDIUM"
    }
  ],
  "epss": {
    "score": 0.0014,
    "percentile": 35.092
  },
  "riskScore": 39,
  "exploitationStatus": "UNKNOWN",
  "affectedSystems": {
    "cpeMatches": [],
    "vendors": ["ibm"],
    "products": ["mq"]
  },
  "references": [
    {
      "url": "https://www.ibm.com/support/pages/node/7157979",
      "source": "",
      "tags": ["vendor-advisory"]
    },
    {
      "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/290259",
      "source": "",
      "tags": ["vdb-entry"]
    }
  ],
  "attackTechniques": [],
  "metadata": {
    "tags": ["CWE-770"],
    "requiresUserInteraction": null,
    "requiresPrivileges": null,
    "attackVector": null,
    "sources": [
      {
        "name": "CVEList",
        "url": "https://github.com/CVEProject/cvelistV5/blob/main/cves/2024/31xxx/CVE-2024-31919.json",
        "lastModified": "2024-08-02T01:59:50.566000+00:00"
      }
    ]
  }
}
